src/components/code-line.component.js

```diff
@@ -130,6 +130,7 @@ export class CodeLine extends Component {
                 CodeTag={Text}
                 codeTagProps={{ style: styles.codeLine }}
                 customStyle={customStyle}
+                fontFamily={fonts.fontCode.fontFamily}
```

Comment:
[`react-native-syntax-highlighter`](https://github.com/conorhastings/react-native-syntax-highlighter) generate by default something like that:
```html
<pre>
   <code>
      ...
   </code>
<pre>
```

* I override the `<code>` tag with a `<Text>` one (`CodeTag` prop), that allow us to scroll normally. 
* I give to this `<Text>` some `codeTagProps` (`styles.codeLine` but seems to doesn't apply `Menlo`, idk why...). 
* And finally, `customStyle` are applied to the `pre` tag.

As there are a `fontFamilly` (and `fontSize`) props, I guess these ones overrides these font style.

I don't remember if I tested to spread the `...fonts.fontCode` within `customStyle` (but I tried a lot of configuration). I will try that during the day and let you know :)